component/src/test/java/org/wso2/extension/siddhi/map/keyvalue/sinkmapper/KeyValueSinkMapperTestCase.java

```diff
@@ -0,0 +1,760 @@
+
+    private static final Logger log = Logger.getLogger(KeyValueSinkMapper.class);
+    private AtomicInteger wso2Count = new AtomicInteger(0);
+    private AtomicInteger ibmCount = new AtomicInteger(0);
+
+    @BeforeMethod
+    public void init() {
+        wso2Count.set(0);
+        ibmCount.set(0);
+    }
+
+    /*
+    * Default keyvalue output mapping
+    */
+    @Test
+    public void keyvalueSinkMapperDefaultTestCase1() throws InterruptedException {
+        log.info("KeyValueSinkMapper-Default TestCase 1");
+        InMemoryBroker.Subscriber subscriberWSO2 = new InMemoryBroker.Subscriber() {
+            @Override
+            public void onMessage(Object msg) {
+
+                HashMap<String, Object> map;
+                switch (wso2Count.incrementAndGet()) {
+                    case 1:
+                        map = new HashMap<>();
+                        map.put("symbol", "WSO2");
+                        map.put("price", 55.6f);
+                        map.put("volume", 100L);
+                        AssertJUnit.assertEquals(map, msg);
+                        break;
+                    case 2:
+                        map = new HashMap<>();
+                        map.put("symbol", "WSO2");
+                        map.put("price", 57.678f);
+                        map.put("volume", 100L);
+                        AssertJUnit.assertEquals(map, msg);
+                        break;
+
+                    case 3:
+                        map = new HashMap<>();
+                        map.put("symbol", "WSO2");
+                        map.put("price", 50f);
+                        map.put("volume", 100L);
+                        AssertJUnit.assertEquals(map, msg);
+                        break;
+
+                    case 4:
+                        map = new HashMap<>();
+                        map.put("symbol", "WSO2#$%");
+                        map.put("price", 50f);
+                        map.put("volume", 100L);
+                        AssertJUnit.assertEquals(map, msg);
+                        break;
+
+                    case 5:
+                        map = new HashMap<>();
+                        map.put("symbol", "WSO2");
+                        map.put("price", 55.6f);
+                        map.put("volume", 100L);
+                        AssertJUnit.assertEquals(map, msg);
+                        break;
+
+                    case 6:
+                        map = new HashMap<>();
+                        map.put("symbol", "IBM");
+                        map.put("price", 32.6f);
+                        map.put("volume", 160L);
+                        AssertJUnit.assertEquals(map, msg);
+                        break;
+
+                    default:
+                        log.error("Received more than expected number of events. Expected maximum : 6," +
+                                "Received : " + wso2Count.get());
+                        AssertJUnit.fail();
```

Comment:
During the code review, it was suggested to specify the reason for the Assertion failure. Could you please fix it?